### PR TITLE
Add future payments component

### PR DESCRIPTION
**Pull request by FadhlanR**
Tickets: CS-5010, CS-5008

Screen Shot 2022-12-22 at 15 13 51

### CODE REVIEW

File: packages/safe-tools-client/app/components/future-payments-list/index.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  height: 100%;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.no-payments-section .no-payments-title {
+  font: 700 1.125rem/calc(24 / 18) var(--boxel-font-family);
```
Comment (jurgenwerk) [inline packages/safe-tools-client/app/components/future-payments-list/index.css:20]: Where is this rem calculation coming from?

File: packages/safe-tools-client/app/components/future-payments-list/index.gts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    //TODO: implement add fund action
+  }
+
+  <template>
+    <BoxelActionContainer 
+      class="future-payments-list"
+      as |Section ActionChin|>
+      {{#if (lte this.futurePayments.length 0)}}
+        <Section class="no-payments-section">
+          <div class="no-payments-title">Scheduled your first payment</div>
```
Comment (jurgenwerk) [inline packages/safe-tools-client/app/components/future-payments-list/index.gts:37]: Would be good to use more BEM-like classes here, for example:

Block: `future-payments-list`
Elements: `future-payments-list__no-payments-section`, `future-payments-list__no-payments-title`, `future-payments-list__no-payments-description`

Comment (jurgenwerk) [reply packages/safe-tools-client/app/components/future-payments-list/index.gts:37]: Then you can remove nested CSS selectors like `.no-payments-section .no-payments-title` and use `future-payments-list__no-payments-title` instead

File: packages/safe-tools-client/app/templates/schedule.hbs [hunk 3]
```diff
@@ -54,7 +54,11 @@
   <section class='safe-tools__dashboard-schedule-content'>
     <ScheduleCollapsePanel @open={{not this.wallet.isConnected}} />
     <div class='temporary-form-container'>
```
Comment (jurgenwerk) [inline packages/safe-tools-client/app/templates/schedule.hbs:56]: Probably a good time to get rid of the "temporary form container" now that we have all 3 panels (form and both lists)

**Review by jurgenwerk — CHANGES_REQUESTED** [review]

**Comment by VitaliKirpu** [comment]
keep rocking guys!

File: packages/safe-tools-client/app/templates/schedule.hbs (hunk 3 above)
Comment (FadhlanR) [reply packages/safe-tools-client/app/templates/schedule.hbs]: Updated

File: packages/safe-tools-client/app/components/future-payments-list/index.gts (hunk 2 above)
Comment (FadhlanR) [reply packages/safe-tools-client/app/components/future-payments-list/index.gts]: Updated

File: packages/safe-tools-client/app/components/future-payments-list/index.css (hunk 1 above)
Comment (FadhlanR) [reply packages/safe-tools-client/app/components/future-payments-list/index.css:20]: I got it from this class `.boxel-action-container-section-title`

**Pull request merged**